src/plugins/kibana_usage_collection/server/collectors/management/schema.ts

```diff
@@ -6,10 +6,7 @@
  * Side Public License, v 1.
  */
 
-import { MakeSchemaFrom } from 'src/plugins/usage_collection/server';
-import { UsageStats } from './types';
-
-export const stackManagementSchema: MakeSchemaFrom<UsageStats> = {
```

Comment:
If we bring these lines back, we can keep the schema defined on a separate file (mind the importance of `MakeSchemaFrom<UsageStats>`